eventbrite.py

```diff
@@ -0,0 +1,422 @@
+        error_detail = error_data.get('error_detail', {})
+        return {
+            "error": error_data.get('error', 'Unknown Error'),
+            "error_description": error_data.get('error_description', 'An error occurred'),
+            "error_detail": error_detail,
+            "status_code": resp.status
+            }
+    data = resp.data.decode('utf-8')
+    print({"message": "Event: {} published successfully".format(event_id), "status_code": resp.status})
+    return json.loads(data)
+
+def create_ticket_class(http, eventbrite_api_token, event_id, ticket_details):
+    url = f"{EVENTBRITE_API_BASE_URL}events/{event_id}/ticket_classes/"
+    headers = {
+    'Authorization': eventbrite_api_token,
+    'Content-Type': 'application/json'
+    }
+
+    # Make the POST request
+    resp = http.request('POST', url, headers=headers, body=ticket_details)
+    
+    # Check the status code to ensure a successful request
+
+
+    if resp.status != 200:
+        error_data = json.loads(resp.data.decode('utf-8'))
+        error_detail = error_data.get('error_detail', {})
+        return {
+            "error": error_data.get('error', 'Unknown Error'),
+            "error_description": error_data.get('error_description', 'An error occurred'),
+            "error_detail": error_detail,
+            "status_code": resp.status
+            }
+
+    data = resp.data.decode('utf-8')
+    return json.loads(data)
+    
+def get_ticket_class_by_id(http, eventbrite_api_token, event_id, ticket_class_id):
+    url = f"{EVENTBRITE_API_BASE_URL}events/{event_id}/ticket_classes/{ticket_class_id}/"
+    headers = {
+    'Authorization': eventbrite_api_token,
+    }
+
+    # Make the GET request
+    resp = http.request('GET', url, headers=headers)
+
+    # Check the status code to ensure a successful request
+    if resp.status != 200:
+        print(f"HTTP Error: {resp.status}")
+
+    data = resp.data.decode('utf-8')
+    print(data)
+    return data
+
+def update_ticket_class(http, eventbrite_api_token, event_id, ticket_class_id, ticket_details):
+    url = f"{EVENTBRITE_API_BASE_URL}events/{event_id}/ticket_classes/{ticket_class_id}/"
+    headers = {
+        'Authorization': eventbrite_api_token,
+        'Content-Type': 'application/json'
+    }
+    
+    # Make the POST request
+    resp = http.request('POST', url, headers=headers, body=ticket_details)
+    
+    if resp.status != 200:
+        error_data = json.loads(resp.data.decode('utf-8'))
+        error_detail = error_data.get('error_detail', {})
+        return {
+            "error": error_data.get('error', 'Unknown Error'),
+            "error_description": error_data.get('error_description', 'An error occurred'),
+            "error_detail": error_detail,
+            "status_code": resp.status
+        }
+    data = resp.data.decode('utf-8')
+    return json.loads(data)
+
+def quick_create_event(http, eventbrite_api_token, organization_id, title, description, cost_cents=None, publish=True):
```

Comment:
missing `date_start` (DateTime object) and `duration_hours` (float) arguments. You can use those 2 values to calculate the end datetime.
```py
from datetime import timedelta
date_start + timedelta(hours=duration_hours)
```